deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py

```diff
@@ -0,0 +1,339 @@
+import argparse
+import os
```

Comment:
All new modules should follow the pep8 code style, typing, include docstring for all public methods, and include unit testing included in the development. Some issues to fix:
- Module has not docstrings
- Import block is un-sorted or un-formatted
- Other warnings of style detected
```
pycodestyle deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py         
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:17:39: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:26:1: E302 expected 2 blank lines, found 1
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:28:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:39:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:54:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:57:33: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:77:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:80:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:102:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:113:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:123:90: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:148:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:151:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:167:44: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:171:21: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:181:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:186:44: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:205:44: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:213:102: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:275:25: E128 continuation line under-indented for visual indent
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:276:25: E128 continuation line under-indented for visual indent
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:278:25: E128 continuation line under-indented for visual indent
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:279:25: E128 continuation line under-indented for visual indent
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:281:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:285:59: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:286:82: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:292:8: W291 trailing whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:299:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:305:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:319:1: W293 blank line contains whitespace
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:334:5: E265 block comment should start with '# '
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:335:5: E265 block comment should start with '# '
deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py:336:5: E265 block comment should start with '# '
```
 
Please these warnings, ensuring your files match the correct format. Remember, the maximum line length for the project is 120 characters.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5622/commits/30888eeaec38bda833044aafd49e7665ed4fc6e2